dnf/repo.py

```diff
@@ -444,6 +444,12 @@ def download_size(self):
 
 class MDPayload(dnf.callback.Payload):
 
+    def __init__(self, progress):
+        super(MDPayload, self).__init__(progress)
+        self._text = ""
+        self._download_size = 0
+        self.fm_running = False
```

Comment:
`fm`? What does `fm` mean?

Reply:
I did not create the attribute, I only set the attribute during init phase.

Reply:
Looks like it could be `fastest_mirror_running`. Can you rename it, please?

Reply:
@mhatina Added new commit with requested change.